src/core/src/op/eye.cpp

```diff
@@ -81,3 +110,73 @@ std::shared_ptr<ov::Node> ov::op::v9::Eye::clone_with_new_inputs(const ov::Outpu
         throw ov::Exception("Eye has incorrect input number: " + std::to_string(new_args.size()));
     }
 }
+
```

Comment:
I'm not sure, but from the spec, looks like Eye-9 may have the case that new_args.size()==1 or 2, but from clone_with_new_inputs() function seems this cases are not allowed? I think spec better align with implement~

Reply:
Eye-9 spec was updated with Eye op shell implementation.
I can see three inputs marked as Required in the spec version on master.
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/generation/Eye_9.md
Are we refer to the same version of the spec?

Reply:
oh, I referred to the PR that the JIRA ticket(82299) mentioned, looks like it has been updated...
based on this master branch version, it looks alright here